tools/istio-iptables/pkg/config/config.go

```diff
@@ -41,7 +41,7 @@ type Config struct {
 	OutboundPortsExclude    string        `json:"OUTBOUND_PORTS_EXCLUDE"`
 	OutboundIPRangesInclude string        `json:"OUTBOUND_IPRANGES_INCLUDE"`
 	OutboundIPRangesExclude string        `json:"OUTBOUND_IPRANGES_EXCLUDE"`
-	KubevirtInterfaces      string        `json:"KUBEVIRT_INTERFACES"`
+	KubeVirtInterfaces      string        `json:"KUBE_VIRT_INTERFACES"`
```

Comment:
Is this backward compatible ? Did we ever use the old name ?

Reply:
The config is incredibly confusing but I am almost certain the json tags here are just for logging. The real config uses `viper` to read, which has always had the separator https://github.com/istio/istio/blob/4ca427a5f1483e4280dd8c7f45c83ff0d7eca81a/tools/istio-iptables/pkg/constants/constants.go#L91.

It would probably be good to clean the config up here, as I cannot wrap my head around it easily

Reply:
If we are sure it was never used - sounds good. It's the type of changes that may have subtle upgrade breakages if someone ever had to set it.